uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/form/components/form-sections/leave-type-wizard-form-settings.html

```diff
@@ -0,0 +1,52 @@
+<div class="row">
+  <div class="col-md-3">
+    <ul class="nav nav-pills nav-stacked">
+      <li ng-repeat="settingsTab in form.settingsTabs"
+        ng-class="{ active: settingsTab.active }"
+        ng-click="form.openSettingsTab(settingsTab.name)">
+        <a href="#">
+          {{settingsTab.label}}
+        </a>
+      </li>
+    </ul>
+  </div>
+  <div class="col-md-9">
+    <div class="row" ng-repeat="field in form.getSettingsTabFields()">
+      <div class="col-md-3">
+        <label>
+          {{field.label}}
+          <help-text ng-if="field.helpText">
+            <p class="respect-line-breaks">{{field.helpText}}</p>
+          </help-text>
+        </label>
+      </div>
+      <div class="col-md-9">
+        [template of the field]
+      </div>
+    </div>
+    <hr/>
+    <div class="row">
+      <div class="col-md-12">
+        <div class="btn btn-secondary pull-right"
+          ng-if="!form.isOnLastSettingsTab"
+          ng-click="form.openSettingsTab(form.settingsTabs[form.activeSettingsTabIndex + 1].name)">
+          Next
+        </div>
+        <div class="btn btn-primary pull-right"
+          ng-if="form.isOnLastSettingsTab">
+          Save
+        </div>
+        <div class="btn btn-secondary"
+          ng-if="form.isOnFirstSettingsTab"
+          ng-click="form.openSection('general')">
```

Comment:
Why the need to hardcode the section name? What if it changes in the future? Can't we have
```html
ng-click="form.openPrevSection()">
```

Reply:
In case of sections I think it is OK to have them hardcoded.

Reply:
Given how experimental this wizard is and the possibility that the spec could still change, especially once you'll submit the first working version, i strongly suggest to not hardcode anything and make the system flexible. You're in the early stage of development, it will cost you less to do it now then to do it at the end

Reply:
Done.